lib/identity_config.rb

```diff
@@ -1,4 +1,7 @@
 class IdentityConfig
+  GIT_SHA = `git rev-parse HEAD`.chomp
+  GIT_TAG = `git tag --points-at HEAD`.chomp.split("\n").first
```

Comment:
we also have the `public/api/deploy.json` file that gets dropped in by our deploy process, WDYT about reading that? It's probably a little more delicate than this but 🤷 

We also use the `branch` there instead of `tag`? I feel like that would be more useful for lower envs/personal envs. 

```suggestion
  deploy_info = JSON.parse(Rails.root.join('public/api/deploy.json').read, symbolize_names: true)
  GIT_STA = deploy_info[:git_sha].freeze
  GIT_BRANCH = deploy_info[:git_branch].freeze
```

Reply:
I considered that, but went with git directly since the file wouldn't work well locally. I'm open to using the file though.

I used tag as a more user-friendly way to compare deploys since it'll be `2021-08-24T195506` and `2021-08-17T195506` instead of a bunch of hexadecimal and branch names will be identical in most environments between deploys.

Reply:
Only prod will have tags? Can we fall back to branches for personal envs?

Reply:
Also, WDYT of taking the first 8 chars of the SHA? Just to keep log lines a little smaller and easier to skim?

Reply:
added branch fallback in 4210a8b96

shorter SHA is a good call, the `--short=N` argument should work here, updated in c8b9e38e3

https://git-scm.com/docs/git-rev-parse#Documentation/git-rev-parse.txt---shortlength